bitnami/kubeapps/README.md

```diff
@@ -330,102 +741,61 @@ helm install --name kubeapps --namespace kubeapps bitnami/kubeapps --set rbac.cr
-- If you want to upgrade to this version from a previous one installed with Helm 3, you shouldn't face any issues related to the new `apiVersion`. Due to the PostgreSQL major version bump, it's necessary to remove the existing statefulsets:
-
-> Note: The command below assumes that Kubeapps has been deployed in the kubeapps namespace using "kubeapps" as release name, if that's not the case, adapt the command accordingly.
-
-```console
-$ kubectl delete statefulset -n kubeapps kubeapps-postgresql-master kubeapps-postgresql-slave
-```
-
-**Useful links**
-
-- https://docs.bitnami.com/tutorials/resolve-helm2-helm3-post-migration-issues/
-- https://helm.sh/docs/topics/v2_v3_migration/
-- https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/
-
-### Upgrading to 2.0
-
-Kubeapps 2.0 (Chart version 4.0.0) introduces some breaking changes:
-
- - Helm 2 is no longer supported. If you are still using some Helm 2 charts, [migrate them with the available tools](https://helm.sh/docs/topics/v2_v3_migration/). Note that some charts (but not all of them) may require to be migrated to the [new Chart specification (v2)](https://helm.sh/docs/topics/charts/#the-apiversion-field). If you are facing any issue managing this migration and Kubeapps, please open a new issue!
- - MongoDB&reg; is no longer supported. Since 2.0, the only database supported is PostgreSQL.
- - PostgreSQL chart dependency has been upgraded to a new major version.
-
-Due to the last point, it's necessary to run a command before upgrading to Kubeapps 2.0:
+### Upgrading to chart version 7.0.0
 
-> Note: The command below assumes that Kubeapps has been deployed in the kubeapps namespace using "kubeapps" as release name, if that's not the case, adapt the command accordingly.
+In this release, no breaking changes were included in Kubeapps (version 2.3.2). However, the chart adopted the standardizations included in the rest of the charts in the Bitnami catalog.
 
-```bash
-kubectl delete statefulset -n kubeapps kubeapps-postgresql-master kubeapps-postgresql-slave
-```
+Most of these standardizations simply add new parameters that allow to add more customizations such as adding custom env. variables, volumes or sidecar containers. That said, some of them include breaking changes:
 
-After that you should be able to upgrade Kubeapps as always and the database will be repopulated.
+- Chart labels were adapted to follow the [Helm charts standard labels](https://helm.sh/docs/chart_best_practices/labels/#standard-labels).
+- `securityContext.*` parameters are deprecated in favor of `XXX.podSecurityContext.*` and `XXX.containerSecurityContext.*`, where XXX is placeholder you need to replace with the actual component(s). For instance, to modify the container security context for "kubeops" use `kubeops.podSecurityContext` and `kubeops.containerSecurityContext` parameters.
```

Comment:
I expected here to read what steps a user will need to take to upgrade? Or if I update the security context options (or haven't used them) will it just work? I don't think the changed labels should break an upgrade (which is not dependent on those labels for deployment/pod relationships etc.)?

Reply:
The labels do break the upgrade since fields such as `spec.selector.matchLabels` are immutable in a deployment.

> I expected here to read what steps a user will need to take to upgrade?

They're mentioned in the paragraphs above (see "Error while upgrading the Chart")